### ISSUE TITLE
Debugging tests that import assertEquals (and maybe others) breaks in colors.ts on line 54 unexpectedly

### ISSUE THREAD

**Issue by frankhale** [denoland/vscode_deno#896]
**Describe the bug**

When debugging tests in VSCode that import assertEquals the debugger always breaks on line 54 of colors.ts despite not having any explicit breakpoint set for that file/line. 

**To Reproduce**

Use Visual Studio Code to debug the following test. Notice that the debugger breaks on line 54 of colors.ts

```typescript
import { assertEquals } from "https://deno.land/std@0.199.0/assert/assert_equals.ts";

Deno.test("hello", () => {
  assertEquals("hello", "hello");
});
```

**Expected behavior**

I expect that no breakpoints are hit that are not set by me. I should not be hitting breakpoints in library code that is outside the scope of what I'm trying to test, unless I've explicitly set it.

**Screenshots**

![image](https://github.com/denoland/vscode_deno/assets/190154/6df9526a-3765-42f3-9ee4-0e683c53d96c)

![image](https://github.com/denoland/vscode_deno/assets/190154/70cdde2c-69ea-4385-8698-bbaa89364f1d)

![image](https://github.com/denoland/vscode_deno/assets/190154/94a4198e-cc35-422b-87b7-c5e0a2ee898f)

![image](https://github.com/denoland/vscode_deno/assets/190154/a1a5a511-410a-4529-8c8a-883bb42ed0d3)

**Versions**

VSCode 1.81.1
Deno 1.36.3
Deno VSCode extension: 3.20.0

**Comment by Leokuma** [comment]
May be fixed by #893.

**Comment by bartlomieju** [comment]
This is actually expected behavior with `--inspect-brk` flag - it puts a breakpoint in the "first" file of user code, where "first" means the deepest dependency on the left hand side of the module graph (https://v8.dev/features/top-level-await#module-execution-order). As mentioned above this will be fixed by #893 which will cause "Debug" to use `--inspect-wait` instead of `--inspect-brk`. A new version of the extension should be released in a week. I'm gonna close this issue for now.
